StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/AccessoryButton.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  AccessoryButton.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 5/28/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+final class AccessoryButton: UIButton {
```

Comment:
Why override UIButton? I think by default, as a matter of best practice, we should avoid subclassing UIKit stuff and prefer composition.  In this case, this doesn't really act like a subclass of UIButton. e.g. users can call `AccessoryButton.buttonWithType` on it, but what does that mean here?

Reply:
Yeah agree with preferring composition. In this case it's mainly so we can override the intrinsic content size so the button is sized correctly w.r.t to the image offset. Open to suggestions for alternative approaches, could maybe change this to sub-class `UIView` or `UIControl` like we do for `CircularButton` but that is still UIKit.